Ambee Destination docs (#5262)
segmentio/segment-docs: src/connections/destinations/catalog/actions-ambee/index.md

The write key is a unique identifier for each Source. It lets Segment
know which Source is sending the data and which destinations should
receive that data.

To find a write key, you first need to create a non-cloud Source such as
a website, server, or mobile source.

Then, in the Source, navigate to **Settings** > **API Keys**.

### API Key

To start working with Ambee as your destination, you'll need
Ambee's API Key. Sign up for Ambee [here](https://auth.ambeedata.com/users/register?redirectUrl=https://api-dashboard.getambee.com){:target="_blank"}.

Once you are signed in, you will get your limited-period API key on the
dashboard's homepage. If your use case requires data in bulk, you'll
need to subscribe to Ambee's [enterprise
plan](https://www.getambee.com/pricing){:target="_blank"}. During
subscription, you'll need to specify your choice of API: **air quality**
and/or **pollen**.

After subscription, you can use Ambee's air quality and/or pollen data
by pasting your API key under **the API Key section** in **Mapping**.

## Ambee Destination Mapping

### Ambee Air Quality Subscription

Ambee's air quality subscription helps you personalize and position your

> messaging via emails, SMS, and push notifications to improve user

Reviewer: ```suggestion
messaging with emails, SMS, and push notifications to improve user
```

  | Unhealthy           | 151-200         | When air quality is in this range, everyone who is active outdoors may experience effects. Members of sensitive groups are likely to experience more serious effects.   |
  | Very unhealthy      | 201-300         | When air quality is in this range, it is expected that there will be widespread effects among general population and more serious effects in members of sensitive groups.  |
  | Hazardous           | 301-500         | Air quality in this range triggers health warnings of emergency conditions. The entire population is more likely to be affected by serious health effects.          
  
### Ambee Pollen Subscription

Ambee's pollen subscription helps you contextualize and hyper-target
your marketing and advertising campaigns with emails, SMS, and push
notifications to improve user engagement with the help of pollen
triggers for locations across the world. You can also personalize your
ad content based on the user's geolocation, for example:

- Advertise anti-allergy tissues based on high pollen condition

- Promote OTC medicines to allergy sufferers

> The risk levels below are recommended by NAB to understand the risks

Reviewer: ```suggestion
NAB recommends the risk levels below to understand the risks
```

> they are exposed to, and it helps them take preventative measures to

Reviewer: ```suggestion
users are exposed to, and help them take preventative measures to
```

> reduce their exposure to pollen. Ambee uses the NAB-compliant index to

Reviewer: ```suggestion
reduce exposure to pollen. Ambee uses the NAB-compliant index to
```
